### PR TITLE
Heading changes

### PR DESCRIPTION
**Pull request by acev0010**
Test

### CODE REVIEW

File: src/_includes/partials/head.njk [hunk 1]
```diff
@@ -25,6 +25,18 @@
...
+	<meta name="author" content="{{ header[locale].governmentOfCanada }}" />
+	<meta name="dcterms.title" content="{{ title }}" />		
+	<meta name="dcterms.description" content="{{  description }}" />
+	<meta name="dcterms.creator" content="{{ header[locale].governmentOfCanada }}">
+	<meta name="dcterms.language" content="{{ locale }}" />
+	<meta name="dcterms.modified" content="{{ page.date | postDate }}" >
+	<meta name="dcterms.created" content="{{ src.gitCreatedDate }}" >
+	<meta name="dcterms.subject" content="{% if subject %}{{ tagList.subjects[locale][subject] }}; 
+
+	{% endif %}{% for item in tags %}{{ tagList.tags[locale][item] }}; {% endfor %}" />
```
Comment (shawnthompson) [inline src/_includes/partials/head.njk:39]: Put this on the same line

File: src/_includes/partials/head.njk [hunk 2]
```diff
@@ -25,6 +25,18 @@
...
 
+	<!-- Meta tags -->
+
+	<meta name="author" content="{{ header[locale].governmentOfCanada }}" />
+	<meta name="dcterms.title" content="{{ title }}" />		
+	<meta name="dcterms.description" content="{{  description }}" />
+	<meta name="dcterms.creator" content="{{ header[locale].governmentOfCanada }}">
+	<meta name="dcterms.language" content="{{ locale }}" />
+	<meta name="dcterms.modified" content="{{ page.date | postDate }}" >
+	<meta name="dcterms.created" content="{{ src.gitCreatedDate }}" >
```
Comment (shawnthompson) [inline src/_includes/partials/head.njk:36]: We should be able to pull this in without the changes to the .eleventy file

File: .eleventy.js
Comment (shawnthompson) [inline .eleventy.js:1]: Might not need the changes in here.

**Review by shawnthompson — CHANGES_REQUESTED** [review]

**Comment by acev0010** [comment]
last commit fixes #344

**Comment by shawnthompson** [comment]
Closes #466 #281 #344

**Pull request merged**